.editorconfig

```diff
@@ -15,5 +15,6 @@ dotnet_diagnostic.RS1017.severity = none
 dotnet_diagnostic.RS1024.severity = error
 dotnet_diagnostic.RS1025.severity = error
 dotnet_diagnostic.RS1026.severity = error
+dotnet_diagnostic.TI11409.severity = none
```

Comment:
Whats this?

Reply:
If you have the Tiobe TICS plugin installed, it will inject an Analyzer that will complain about the tabs in EVERY line of code.
So, I disabled that check here, as you explicitly want tabs instead of spaces.

Tiobe creates Diagnostics prefixed with TI, the tabs rule happens to be 11409, apparently.